### PR TITLE
doc/md: mssql guide

### PR DESCRIPTION
**Pull request by hilakashai**

### CODE REVIEW

**Review by giautm — APPROVED** [review]

File: doc/md/guides/mssql.mdx [hunk 1]
```diff
@@ -0,0 +1,618 @@
+---
+id: getting-started-mssql
+title: Manage MSSQL with Atlas
```
Comment (rotemtam) [inline doc/md/guides/mssql.mdx:3]:
```suggestion
title: Manage SQL Server with Atlas
```

Comment (rotemtam) [reply doc/md/guides/mssql.mdx:3]: Same for ClickHouse

File: doc/md/guides/mssql.mdx [hunk 2]
```diff
@@ -0,0 +1,618 @@
...
+slug: /guides/mssql
+tags: [mssql]
+---
+
+import InstallationInstructions from '../components/_installation_instructions.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/) (MSSQL) is a powerful relational database management
```
Comment (rotemtam) [inline doc/md/guides/mssql.mdx:12]:
```suggestion
[Microsoft SQL Server](https://www.microsoft.com/en-us/sql-server/) is a powerful relational database management
```

File: doc/md/guides/mssql.mdx [hunk 3]
```diff
@@ -0,0 +1,618 @@
...
+```shell
+atlas schema inspect -u "sqlserver://SA:P@ssw0rd0995@localhost:1433?database=master&mode=database" -w
+```
+
+![mssql-inspect](https://atlasgo.io/uploads/mssql/images/schema-inspect-hcl.png)
+
+## Declarative Migrations
+
+The declarative approach allows users to manage their database schemas by simply defining the desired end schema, and
+Atlas provides a safe way to alter the database to get there. Let's see this in action.
```
Comment (rotemtam) [inline doc/md/guides/mssql.mdx:167]: Something broken about this sentence

File: doc/md/guides/mssql.mdx [hunk 4]
```diff
@@ -0,0 +1,616 @@
...
+
+```shell
+atlas schema inspect -u "sqlserver://SA:P@ssw0rd0995@localhost:1433?database=master&mode=database" -w
+```
+
+![mssql-inspect](https://atlasgo.io/uploads/mssql/images/schema-inspect-hcl.png)
+
+## Declarative Migrations
+The declarative approach lets users manage schemas by defining the desired state. Atlas then safely alters the database
+to match. Let's see this in action.
```
Comment (rotemtam) [inline doc/md/guides/mssql.mdx:166]:
```suggestion
## Declarative Migrations

The declarative approach lets users manage schemas by defining the desired state of the database as code.
Atlas then inspects the target database and calculates an execution plan to reconcile the difference between the desired and actual states.  Let's see this in action.
```

Do the same for C.H

**Review by rotemtam — APPROVED** [review]
See small suggestion on the declarative migration opening. Other than that, LGTM

**Pull request merged**